### PR TITLE
BZ1698198--remove oc label command to remove daemonset label

### PR DESCRIPTION
**Pull request by jhoyt-rh**

### CODE REVIEW

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -731,13 +731,13 @@ Linux Atomic Host.
 When upgrading  {product-title}, you must upgrade the set of nodes where
 GlusterFS pods are running.
 
-Special consideration must be taken when upgrading these nodes, as `drain` and
+Use special consideration when upgrading these nodes, as `drain` and
```
Comment (bergerhoffer) [inline upgrading/automated_upgrades.adoc]: This phrasing of "Use special consideration" sounds odd to me. Consider rewording, maybe something like "Use caution when..."?

**Comment by jhoyt-rh** [comment]
This PR is for BZ 1698198.

File: upgrading/automated_upgrades.adoc (hunk 1 above)
Comment (jhoyt-rh) [reply upgrading/automated_upgrades.adoc]: @ bergerhoffer. agree. I changed it to "use care"

**Comment by bergerhoffer** [comment]
@jhoyt-rh There was mention of the specifics for the playbook in that BZ, is that something that should be covered as well by this PR?

**Comment by jhoyt-rh** [comment]
@bergerhoffer Good point about the playbooks. I'm doing some research on it, and expect to update soon.

**Comment by jhoyt-rh** [comment]
Closing this second PR related to BZ1698198 also. The BZ has been around and in review for a long time, and additional changes were made. Am opening a new PR.

**Pull request closed without merging**